app/include/zmk/events/battery_state_changed.h

```diff
@@ -10,8 +10,40 @@
 #include <zmk/event_manager.h>
 
 struct zmk_battery_state_changed {
-    // TODO: Other battery channels
+    
```

Comment:
Yeah I know... I meant to go back into the driver and find out for sure what units were being output, but I can still do that. There are only a couple of channels I am using atm, which is why I have not deeply investigated the others. Wanted to get other people's eyes on this sooner rather than later.